### ISSUE TITLE
Apply CoderSchool Select submition

### ISSUE THREAD

**Issue by ngominhtrint** [ngominhtrint/tipCalculator#2]
Hello Harley,
This is my update for Select program.

Chang log:
1. Auto layout for all screen.
2. Add game screen for lucky tip money.
3. Game screen with spin view. It spin to get random tip rate.

/cc @coderschoolreview

**Comment by harley** [comment]
Woohoo thanks for the submission. Could you go back to your application at http://apply.coderschool.vn/members/login/?next=/ and use the Prework Submission form on the website?

**Comment by ngominhtrint** [comment]
I've done @harley . Please check it :)

**Comment by coderschoolreview** [comment]
Yes I see it in the system now (and this reply comes from our new system)! We'll review your work and be in touch very soon (today).

**Comment by ngominhtrint** [comment]
Thank you Harley :)
